x-pack/plugins/cloud_security_posture/server/routes/benchmarks/benchmarks.ts

```diff
@@ -193,4 +198,8 @@ export const benchmarksInputSchema = rt.object({
    * The number of objects to include in each page
    */
   per_page: rt.number({ defaultValue: DEFAULT_BENCHMARKS_PER_PAGE, min: 0 }),
+  /**
+   * Benchmark filter
+   */
+  benchmark_filter: rt.maybe(rt.string()),
```

Comment:
`benchmark_filter` is a little generic IMO, consider using a more explicit name like `benchmark_name`

Reply:
done